### PR TITLE
[devtools-translate] Portuguese - What's New in DevTools (Chrome 110)

### LINKED ISSUE #5279: [devtools-translate] Portuguese - What's New in DevTools (Chrome 110)
Post: https://developer.chrome.com/blog/new-in-devtools-110/
Translated by: Feb-23

Translator: @alvarocamillont
Reviewers: @alvarocamillont, @khaosdoctor

### PR DESCRIPTION
**Pull request by alvarocamillont**
Fixes #5279

Changes proposed in this pull request:

Add portuguese translations for What's New in DevTools M110

### CODE REVIEW

File: site/pt/_partials/devtools/whats-new.md [hunk 1]
```diff
@@ -4,19 +4,17 @@ Consulte a versão em inglês do <a href="/tags/new-in-devtools/" translate="no"
...
-* [Automatic in-place pretty print](/pt/blog/new-in-devtools-110/#pretty-print)
-* [Better syntax highlight and inline preview for Vue, SCSS and more](/pt/blog/new-in-devtools-110/#highlight)
-* [Ergonomic and consistent Autocomplete in the Console](/pt/blog/new-in-devtools-110/#console)
-* [Miscellaneous highlights](/pt/blog/new-in-devtools-110/#misc)
- #}
-
+### Chrome 110 {: #chrome110 }
+
+* [Limpeza do painel Desempenho ao recarregar](/pt/blog/new-in-devtools-110/#perf)
+* [Atualizações do gravador](/pt/blog/new-in-devtools-110/#recorder)
```
Comment (thiagowfx) [inline site/pt/_partials/devtools/whats-new.md:10]: Gravador

(be consistent with the wording below which capitalizes the first letter)

File: site/pt/_partials/devtools/whats-new.md [hunk 2]
```diff
@@ -4,19 +4,17 @@ Consulte a versão em inglês do <a href="/tags/new-in-devtools/" translate="no"
...
+* [Visualização e realce do código do fluxo de usuário no Gravador](/pt/blog/new-in-devtools-110/#recorder-code)
+* [Personalize os tipos de seletor de uma gravação](/pt/blog/new-in-devtools-110/#recorder-selector)
+* [Edite o fluxo do usuário durante a gravação](/pt/blog/new-in-devtools-110/#recorder-edit)
+* [Formatação automática localizada](/pt/blog/new-in-devtools-110/#pretty-print)
+* [Melhor destaque de sintaxe e visualização inline para Vue, SCSS e mais](/pt/blog/new-in-devtools-110/#highlight)
```
Comment (thiagowfx) [inline site/pt/_partials/devtools/whats-new.md:15]: mais -> outros

File: site/pt/blog/new-in-devtools-110/index.md [hunk 3]
```diff
@@ -27,78 +26,86 @@ draft: true
...
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/0a301d29d165f17a6eceb1adf91bff0c1c2e07eb #}
 
-Chromium issues: [1101268](https://crbug.com/1101268), [1382044](https://crbug.com/1382044)
+Issues relacionadas: [1101268](https://crbug.com/1101268), [1382044](https://crbug.com/1382044)
 
 
 <!-- ## Recorder updates {: #recorder } -->
-
+## Atualizações do gravador {: #recorder }
```
Comment (thiagowfx) [inline site/pt/blog/new-in-devtools-110/index.md:44]: Gravador

**Review by thiagowfx — APPROVED** [review]

File: site/pt/_partials/devtools/whats-new.md [hunk 4]
```diff
@@ -4,19 +4,17 @@ Consulte a versão em inglês do <a href="/tags/new-in-devtools/" translate="no"
...
- #}
-
+### Chrome 110 {: #chrome110 }
+
+* [Limpeza do painel Desempenho ao recarregar](/pt/blog/new-in-devtools-110/#perf)
+* [Atualizações do gravador](/pt/blog/new-in-devtools-110/#recorder)
+* [Visualização e realce do código do fluxo de usuário no Gravador](/pt/blog/new-in-devtools-110/#recorder-code)
+* [Personalize os tipos de seletor de uma gravação](/pt/blog/new-in-devtools-110/#recorder-selector)
+* [Edite o fluxo do usuário durante a gravação](/pt/blog/new-in-devtools-110/#recorder-edit)
+* [Formatação automática localizada](/pt/blog/new-in-devtools-110/#pretty-print)
```
Comment (khaosdoctor) [inline site/pt/_partials/devtools/whats-new.md:14]:
```suggestion
* [Formatação automática na mesma aba](/pt/blog/new-in-devtools-110/#pretty-print)
```

File: site/pt/blog/new-in-devtools-110/index.md [hunk 5]
```diff
@@ -27,78 +26,86 @@ draft: true
...
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/1a2S1lizzJ5acRMgjtwH.png", alt="Editing during user flow recording.", width="800", height="619" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/1a2S1lizzJ5acRMgjtwH.png", alt="Edição durante a gravação do fluxo do usuário.", width="800", height="619" %}
 
-Chromium issue: [1381971](https://crbug.com/1381971)
+Issue relacionada: [1381971](https://crbug.com/1381971)
 
 
 <!-- ## Automatic in-place pretty print {: #pretty-print } -->
-
+## Formatação automática localizada {: #pretty-print }
```
Comment (khaosdoctor) [inline site/pt/blog/new-in-devtools-110/index.md:78]:
```suggestion
## Formatação automática na mesma aba {: #pretty-print }
```

File: site/pt/blog/new-in-devtools-110/index.md [hunk 6]
```diff
@@ -111,48 +118,59 @@ Chromium issues: [1383453](https://crbug.com/1383453), [1382752](https://crbug.c
...
 
 
 <!-- ## Miscellaneous highlights {: #misc } -->
-
+## ## Outros destaques {: #misc }
 <!-- These are some noteworthy fixes in this release: -->
+Estas são algumas correções dignas de nota nesta versão:
 
 <!-- - A regression issue in DevTools, where it failed to stop at the `debugger` statement in inline scripts, has been resolved. ([1385374](https://crbug.com/1385374)) -->
+- Um problema de regressão no DevTools, onde falhava ao parar na instrução `debugger` em scripts embutidos, foi resolvido. ([1385374](https://crbug.com/1385374))
```
Comment (khaosdoctor) [inline site/pt/blog/new-in-devtools-110/index.md:168]:
```suggestion
- Um problema de regressão no DevTools, onde, ao parar na instrução `debugger` em scripts embutidos, gerava uma falha foi resolvido. ([1385374](https://crbug.com/1385374))
```

**Review by khaosdoctor — COMMENTED** [review]
Small Changes

File: site/pt/_partials/devtools/whats-new.md (hunk 1 above)
Comment (alvarocamillont) [reply site/pt/_partials/devtools/whats-new.md:10]: Done

File: site/pt/_partials/devtools/whats-new.md (hunk 2 above)
Comment (alvarocamillont) [reply site/pt/_partials/devtools/whats-new.md:15]: done

File: site/pt/blog/new-in-devtools-110/index.md (hunk 3 above)
Comment (alvarocamillont) [reply site/pt/blog/new-in-devtools-110/index.md:44]: done

File: site/pt/_partials/devtools/whats-new.md (hunk 4 above)
Comment (alvarocamillont) [reply site/pt/_partials/devtools/whats-new.md:14]: Done

File: site/pt/blog/new-in-devtools-110/index.md [hunk 7]
```diff
@@ -27,78 +26,86 @@ draft: true
 
 
 <!-- ## Clearing Performance Panel on reload {: #perf } -->
-
+## Limpeza do painel Desempenho ao recarregar {: #perf } 
 <!-- The **Performance** panel now clears both the screenshot and trace when you click the **Start profiling and reload page** button. -->
-
+O painel **Desempenho** agora limpa a captura de tela e o rastreamento quando você clica no botão **Iniciar criação de perfil e recarregar a página**.
 <!-- Previously, the **Performance** panel displayed a timeline with screenshots from previous recordings. This made it difficult to see when the actual measurement started. The panel now always navigates to the `about:blank` page first to guarantee that the recording begins with a blank trace. This aligns with the **Performance Insights** panel which already did the same. -->
+Anteriormente, o painel **Desempenho** exibia uma linha do tempo com capturas de tela de gravações anteriores. Isso tornou difícil ver quando a medição real começou. O painel agora sempre navega primeiro para a página `about:blank` para garantir que a gravação comece com um trace em branco. Isso se alinha com o painel **Performance Insights**, que já fez o mesmo.
```
[the remark this answers is not in the document]
Comment (alvarocamillont) [reply site/pt/blog/new-in-devtools-110/index.md:33]: Done

File: site/pt/blog/new-in-devtools-110/index.md (hunk 5 above)
Comment (alvarocamillont) [reply site/pt/blog/new-in-devtools-110/index.md:78]: Done

File: site/pt/blog/new-in-devtools-110/index.md [hunk 8]
```diff
@@ -111,48 +118,59 @@ Chromium issues: [1383453](https://crbug.com/1383453), [1382752](https://crbug.c
...
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/00103b19eec2ba086c608b79ff34b696fe07bb62 #}
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/89f259ddb6c36f486108e0dc9ccb4d4125a04917 #}
 
-Chromium issues: [1399436](https://crbug.com/1399436), [1276960](https://crbug.com/1276960)
+Issues relacionadas: [1399436](https://crbug.com/1399436), [1276960](https://crbug.com/1276960)
 
 
 <!-- ## Miscellaneous highlights {: #misc } -->
-
+## ## Outros destaques {: #misc }
```
[the remark this answers is not in the document]
Comment (alvarocamillont) [reply site/pt/blog/new-in-devtools-110/index.md:163]: Done

File: site/pt/blog/new-in-devtools-110/index.md (hunk 6 above)
Comment (alvarocamillont) [reply site/pt/blog/new-in-devtools-110/index.md:168]: Done

**Comment by alvarocamillont** [comment]
Done!

**Review by thiagowfx — APPROVED** [review]

**Review by andreban — APPROVED** [review]
LGTM, thank you!

**Review by jecfish — APPROVED** [review]

File: site/pt/_partials/devtools/whats-new.md [hunk 9]
```diff
@@ -15,18 +16,15 @@ Consulte a versão em inglês do <a href="/tags/new-in-devtools/" translate="no"
...
-* [Recorder updates](/pt/blog/new-in-devtools-110/#recorder)
-* [View and highlight the code of your user flow in the Recorder](/pt/blog/new-in-devtools-110/#recorder-code)
-* [Customize selector types of a recording](/pt/blog/new-in-devtools-110/#recorder-selector)
-* [Edit user flow while recording](/pt/blog/new-in-devtools-110/#recorder-edit)
-* [Automatic in-place pretty print](/pt/blog/new-in-devtools-110/#pretty-print)
-* [Better syntax highlight and inline preview for Vue, SCSS and more](/pt/blog/new-in-devtools-110/#highlight)
-* [Ergonomic and consistent Autocomplete in the Console](/pt/blog/new-in-devtools-110/#console)
-* [Miscellaneous highlights](/pt/blog/new-in-devtools-110/#misc)
- #}
+* [Limpeza do painel Desempenho ao recarregar](/pt/blog/new-in-devtools-110/#perf)
```
Comment (jecfish) [inline site/pt/_partials/devtools/whats-new.md:19]:
```suggestion
### Chrome 110 {: #chrome110 }

* [Limpeza do painel Desempenho ao recarregar](/pt/blog/new-in-devtools-110/#perf)
```

**Comment by jecfish** [comment]
Passed DevRel review. Over to @sofiayem to approve and merge

**Review by sofiayem — APPROVED** [review]

**Pull request merged**